src/test/java/org/vaadin/example/FlowViewIT.java

```diff
@@ -2,54 +2,55 @@
 package org.vaadin.example;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import com.microsoft.playwright.BrowserType;
+import com.microsoft.playwright.Page;
 import com.microsoft.playwright.Page.GetByRoleOptions;
+import com.microsoft.playwright.Playwright;
 import com.microsoft.playwright.options.AriaRole;
-import com.vaadin.uitest.common.BasePlayWrightIT;
 
-public class FlowViewIT extends BasePlayWrightIT {
+public class FlowViewIT  {
 
     private static final String BASE_URL = "http://localhost:8080/flow";
+    private Page page;
 
-    @Override
-    public String getUrl() {
-        return BASE_URL;
-    }
 
     @BeforeEach
     public void setupTest() throws Exception {
-        super.setupTest();
-        // Login steps
-        fill(page.locator("vaadin-login-form vaadin-text-field"), "admin");
-        fill(page.locator("vaadin-login-form vaadin-password-field"), "admin");
-        click(page.locator("vaadin-login-form vaadin-button"));
+        page = Playwright.create().chromium()
+                .launch(new BrowserType.LaunchOptions()
+                        .setHeadless(System.getProperty("headless") == null || Boolean.getBoolean("headless")))
+                .newContext().newPage();
+        page.setDefaultTimeout(30000);
+        page.navigate(BASE_URL);
+        page.locator("vaadin-login-form vaadin-text-field input").fill("admin");
+        page.locator("vaadin-login-form vaadin-password-field input").fill("admin");
+        page.locator("vaadin-login-form vaadin-button").first().click();
         page.waitForURL(BASE_URL);
-        page.waitForSelector("vaadin-button");
     }
 
     @AfterEach
     public void tearDown() {
         page.close();
+        page.context().close();
```

Comment:
This is not necessary as it is included in `browser().close()`. It doesn't hurt though and makes the intention clear.